package.json

```diff
@@ -3,8 +3,28 @@
   "version": "1.0.0",
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
+  "engines": {
+    "node": ">=6.0.0"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node server/index.js",
+    "test": "exit 0",
+    "lint": "./node_modules/.bin/eslint 'server/**/*js'",
```

Comment:
install eslint globally,  I expect `eslint '**/*js`
Why is this being done one by one. it should just be one command to fix
`    "lint:fix": "prettier-eslint '**/*js --write",`
Why is there so much consideration for json files, where are they coming from?